domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -177,11 +177,11 @@ constructor(
     override suspend fun takePicture(contentResolver: ContentResolver, imageCaptureUri: Uri?) {
         val imageDeferred = CompletableDeferred<ImageCapture.OutputFileResults>()
         val eligibleContentValues = getEligibleContentValues()
-        var outputFileOptions: OutputFileOptions? = null
+        val outputFileOptions: OutputFileOptions
```

Comment:
Does this need an `=` at the end of the line?

Reply:
No. It's set on line 189. All other cases lead to exception being thrown